lib/rules/padded-blocks.js

```diff
@@ -17,6 +17,30 @@ module.exports = function (context) {
         NEVER_MESSAGE = "Block must not be padded by blank lines.";
 
     /**
+     * Retrieves an array of all comments defined inside the given node.
+     * @param {ASTNode} node The AST node.
+     * @returns {ASTNode[]} An array of comment nodes.
+     */
+    function getCommentsInNode(node) {
+        var allComments = context.getAllComments();
```

Comment:
Still curious if there's a way to optimize this. Starting atvthe first comment each time seems inefficient.